web-app/src/pages/Dashboard/container.js

```diff
@@ -61,15 +62,57 @@ const DashboardContainer = Wrapped =>
       };
     };
 
+    setMandatoryEvents = () => {
+      const currYear = new Date().getFullYear();
+      const mandatoryEvent = [
```

Comment:
Take this  `mandatoryEvent` array out of DashboardContainer and put it in a configuration file somewhere.